apps/core-app/src/main/modules/box-tool/addon/preview/index.ts

```
@@ -1,13 +1,18 @@
+import { PreviewAbilityRegistry, previewSdk, registerPreviewAbility } from './preview-registry'
 
-registerDefaultPreviewAbilities(previewAbilityRegistry)
+registerDefaultPreviewAbilities({ register: registerPreviewAbility })
 
-export const previewProvider = new PreviewProvider(previewAbilityRegistry)
+export const previewProvider = new PreviewProvider(previewSdk)
 
-export { PreviewAbilityRegistry, registerPreviewAbility }
+export {
+  PreviewAbilityRegistry,
+  listPreviewAbilityInventory,
```

ESLint

[error] Expected "listPreviewAbilityInventory" to come before "PreviewAbilityRegistry".

(perfectionist/sort-named-exports)